a10_octavia/controller/worker/controller_worker.py

```diff
@@ -283,6 +283,7 @@ def create_load_balancer(self, load_balancer_id):
             constants.TOPOLOGY: topology
         }
         if lb.project_id in CONF.hardware_thunder.devices:
+            store.update({constants.LOADBALANCER: lb})
```

Comment:
https://github.com/a10networks/a10-octavia/pull/155/files#diff-381b26a44ca5f8e3358e77ddab33e198L320

```python
            database_tasks.ReloadLoadBalancer(
                name=sf_name + '-' + constants.RELOAD_LB_AFTER_AMP_ASSOC,
                requires=constants.LOADBALANCER_ID,
                provides=constants.LOADBALANCER)
```

We use this later on to fetch the LB later on from the DB. However, we should just be able to move it up to the flow since we need it there. No reason to store and then overwrite later with the exact same data